shopify/resources/gift_card.py

```diff
@@ -5,3 +5,48 @@ class GiftCard(ShopifyResource):
 
     def disable(self):
         self._load_attributes_from_response(self.post("disable"))
+
+    @classmethod
+    def search(cls, **kwargs):
+        """
+        Search for gift cards matching supplied query
+
+        Args:
+           order: Field and direction to order results by (default: disabled_at DESC)
+           query: Text to search for gift cards
+           page: Page to show (default: 1)
+           limit: Amount of results (default: 50) (maximum: 250)
+           fields: comma-seperated list of fields to include in the response
+        Returns:
+           An array of gift cards.
+        """
+        return cls._build_list(cls.get("search", **kwargs))
+
+    def adjust(self, amount, **kwargs):
+        """
+        Create a new Gift Card Adjustment
+
+        To increase send a positive amount. To reduce provide a negative
+        amount.
+
+        Args:
+            amount: A number (signed) representing the value of the adjustment
+            note: The text of an optional note that can be attached to the
+                  adjustment.
+                  This value is not visible to customers.
+            remote_transaction_ref: The text of an optional reference note
+                  that can be attached to the adjustment.
+                  This value is not visible to customers.
+            remote_transaction_url: An optional reference URL that can be
+                  attached to the adjustment.
+                  This value is not visible to customers.
+        """
+        data = kwargs
+        data['amount'] = amount
+        data['gift_card_id'] = self.id
+        return Adjustments.create(data)
```

Comment:
would it be possible to refactor this to be more similar to https://github.com/Shopify/shopify_python_api/blob/master/shopify/resources/price_rule.py#L5-L7